sklearn/impute.py

```diff
@@ -40,6 +40,18 @@
 ]
 
 
+def _check_inputs_dtype(X, missing_values):
+    """Check that the type of missing_values is in accordance with the
```

Comment:
I would remove this docstring, doesn't add much information or document it fully if you insist this function needs to be documented.